components/list/list-item-mixin.js

```diff
@@ -207,6 +207,18 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				--d2l-list-item-content-text-outline: 2px solid var(--d2l-color-celestine);
 				--d2l-list-item-content-text-outline-offset: 1px;
 			}
+			@supports selector(:has(a, b)) {
```

Comment:
Lessons confirmed they are okay with `:has` not being supported by all users. For our supported browsers, only Firefox ESR would not support it and so those users would see a focus ring on click as well.